packages/apollo-server-core/src/plugin/landingPage/default/index.ts

```diff
@@ -203,9 +209,14 @@ curl --request POST \\
   --url '<script>document.write(window.location.href)</script>' \\
   --data '{"query":"query { __typename }"}'</code>
       </div>
+    ${
+      !!config.embed
```

Comment:
does !! really do anything here (and on the next line)?

Reply:
nope - unnecessary